lib/openxml/builder/element.rb

```diff
@@ -21,17 +20,15 @@ def namespace=(ns)
         self.value = "#{namespace}:#{tag}" if namespace.is_a? String
       end
 
+      def namespaces=(ns)
+        @namespaces = ns if ns.is_a?(Array)
```

Comment:
Why do you check if `ns.is_a?(Array)`? Is it ever not? Can we just assume that it is? Or should we coerce `ns` to an array with `Array(ns)` or allow it to cast itself to an array with `ns = ns.to_ary if ns.respond_to?(:to_ary)`?